### PR TITLE
Added json-schema and rights-ml for POE WG specs

### PR DESCRIPTION
**Pull request by riannella**

### CODE REVIEW

File: refs/biblio.json [hunk 1]
```diff
@@ -2793,6 +2793,26 @@
...
+        ],
+        "href": "https://tools.ietf.org/html/draft-zyp-json-schema",
+        "publisher": "IETF",
+        "status": "Internet-Draft",
+        "rawDate": "2013-01-31",
+        "title": "JSON Schema: core definitions and terminology"
+    },
+     "rights-ml": {
+        "authors": [
+            "IPTC"
```
Comment (tobie) [inline refs/biblio.json]: Seems like this is an acronym for the publisher, no?

**Comment by riannella** [comment]
I've expanded the acronymes

**Comment by tobie** [comment]
Oops, sorry, that's not what I meant. I thought it was weird to put as authors the name of the publisher. What's the rationale for doing that?

**Comment by tobie** [comment]
Mmm. Looks like I missed that in your previous commits too. I'll merge that in now, but it would be nice to get some consistency across those, plus leave the authors filed empty if there are no authors.

**Pull request merged**